src/main/java/ca/bc/gov/chefs/etl/constant/Constants.java

```diff
@@ -654,4 +654,63 @@ public class Constants {
 				"SUB_TOTAL_DIR_CARE_COST_HOURLY_RATE_CONTRACTED",
 		});
 	}
+
+/*
+ * 
+ * ------------------------------------------------- PCD BI PROJECT -----------------------------------------------
+ * 
+ */
+
+ 	/* Decision Log Form */
+	public final static String PCD_DECISION_LOG_PROPERTY = "pcd.decision.log.";
+	public final static String PROPERTIES_PCD_DECISION_LOG = "pcd-decision-log-dir";
+	public final static String PCD_DECISION_LOG_DIR = FileUtil.getDirectoryName(PROPERTIES_PCD_DECISION_LOG);
+	public static final String PCD_DECISION_LOG_UNENCRYPTED_FP = FileUtil.buildDestinationPath(PROPERTIES_PCD_DECISION_LOG, false);
+	public static final String PCD_DECISION_LOG_ENCRYPTED_FP = FileUtil.buildDestinationPath(PROPERTIES_PCD_DECISION_LOG, true);
+
+	public final static String DECISION_LOG_SUBMISSIONS = "DECISION_LOG_SUBMISSIONS";
+	public final static String CHANGE_REQUEST_FILE_UPLOAD = "CHANGE_REQUEST_FILE_UPLOAD";
+	public final static String PCN_NAMES = "PCN_NAMES";
+	public final static String DECISION_LOG_COMMENTS = "DECISION_LOG_COMMENTS";
+	public final static String DECISION_LOG_INITIATIVES = "DECISION_LOG_INITIATIVES";
+	public final static String PRIMARY_CARE_INITIATIVES = "PRIMARY_CARE_INITIATIVES";
```

Comment:
Do we want to follow the MEDIS standards, where everything has a standard prefix (e.g. DECISION_LOG_) and table names are singular? There's just the 3 tables that don't have the prefix.

Reply:
all generated tables now have DECISION_LOG prefix